target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -188,6 +188,9 @@ dlink,dap-1365-a1)
 dlink,dir-859-a1)
 	ucidef_set_led_switch "internet" "WAN" "green:internet" "switch0" "0x20"
 	;;
+dlink,dir-869-a1)
+	ucidef_set_led_switch "led_power_white" "POWER" "white:power" "default-on" "0x20"
+	;;
```

Comment:
So, now we have power-LEDs instead of internet-LEDs?

Reply:
The LEDs on this router are a little different, it has white and orange LEDs, but they shine from the same place. White LED indicates that the device is booted. I think there is no need to define an orange LED here, because when the device is successfully booted, the orange LED is not used.

Reply:
The name power seems appropriate to me, because in dir-869-a1.dts I named the LEDs power_white and power_orange.

Reply:
Power LEDs should be defined in DTS. But apart from that, random config changes without any justification make you look like you don't care about what's correct.

Reply:
I'm sorry if it seemed like a random config change. The previous setting of this file caused both LEDs to flash, I modified /etc/config/system so that the white LED lights up after successful boot and I modified this file accordingly. The red LED is not needed define here because the behaviour in a state other than a successful boot is specified in DTS.

Reply:
@honza96 I don't get it, too... So the LEDs are also controlled by AR8327, but you still need this entry although you've defined it in DTS using GPIOs?!

Reply:
hi, i tried build without this entry and LED works fine. Thank you for the advice.